### DISCUSSION TITLE
Is it possible to set the display of test progress (percentage) and progress bar?

### DISCUSSION THREAD

**Discussion by liby** [cypress-io/cypress#17600] [Questions and Help]
I saw the appearance of Cypress in the third season of 《How to Sell Drugs Online (Fast)》, I noticed a slight difference between the Cypress in this TV series and the one I usually use. 

The difference is that when you run the test,  Cypress in the TV series the test progress, while in my daily use it just shows up as running time.

![image](https://user-images.githubusercontent.com/38807139/128214653-d0ff0302-dadc-46c6-b71b-e8189d9c82df.png)

https://user-images.githubusercontent.com/38807139/128214722-042d2a7b-6414-41ac-8177-3332f48a2498.mp4

But the test progress (percentage) shown in the TV series looks like it's been post-processed(video editing), so I doubt if there's really such a feature, can you tell me the answer?

**Comment by david10-1** [comment]
It's really late in every sense, but I think that was part of the magic of television wwwwww
